### PR TITLE
Decode refactor

### PR DESCRIPTION
**Pull request by ptpham**

### CODE REVIEW

File: src/sheet.js [hunk 1]
```diff
@@ -3,6 +3,24 @@ var Halfedge = require('./halfedge'),
     Bimap = require('./bimap'),
     _ = require('lodash');
 
+function Index() { Bimap.call(this); }
```
Comment (cixzhang) [inline src/sheet.js]: Should this be a separate module?

File: test/paper.js [hunk 2]
```diff
@@ -71,18 +71,19 @@ describe('paper', function() {
...
   var square = new Paper();
+  var squareIndex = square.sheet.index;
   it('can convert halfedges to their indices', function() {
     var interior = square.toIndices(square.interior);
     assert.equivalentTo(interior, [1,2,4,6]);
   });
 
   it('can resolve halfedges from indices', function() {
-    var result = square.resolve([1,2,4,6]);
+    var result = squareIndex.decode([1,2,4,6]);
```
Comment (cixzhang) [inline test/paper.js]: Should we move these tests to sheet?

File: src/sheet.js (hunk 1 above)
Comment (ptpham) [inline src/sheet.js]: Uh ... I feel like this is too small and not general enough to be in a separate module. XD

File: test/paper.js (hunk 2 above)
Comment (ptpham) [inline test/paper.js]: I'm working on this.

**Comment by cixzhang** [comment]
Looks good :D

**Pull request merged**